library/teamspeak

```diff
@@ -2,6 +2,10 @@ Maintainers: Niels Werensteijn <niels.werensteijn@teamspeak.com> (@nwerensteijn)
              Maximilian Münchow <maximilian.muenchow@teamspeak.com> (@muenchow)
 GitRepo: https://github.com/TeamSpeak-Systems/teamspeak-linux-docker-images
 
-Tags: 3.1, 3.1.0, latest
+Tags: 3.1, 3.1.1, latest
+GitCommit: 0acf85c990d8289ab5b4c51c1583df4437271995
+Directory: 3.1.1
+
+Tags: 3.1.0
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
The reason we kept the previous there, was that we intended to update the version with possible changes to the dockerfile/entrypoint. But you are right, spamming the  supported section and the build server with every minor version, isn't the way to go.
We will improve that with the next version.